### DISCUSSION TITLE
What should I replace `theme()` with?

### DISCUSSION THREAD

**Discussion by LeBenLeBen** [windicss/windicss#103] [Q&A] [answered]
I moved a Tailwind project to Windi and I have an issue with the following declaration:

```css
border-color: theme('colors.alt.500') transparent transparent;
```

Since `theme` doesn’t exist, I tried to do it differently with `@apply`:

```css
@apply border-alt-500;
border-right-color: transparent;
border-bottom-color: transparent;
border-left-color: transparent;
```

But this doesn’t work, the `@apply  border-alt-500;` is somehow moved to the end of the declaration in the built CSS and therefor overriding the others:

```css
border-right-color: transparent;
border-bottom-color: transparent;
border-left-color: transparent;
--tw-border-opacity: 1;
border-color: rgba(158, 134, 172, var(--tw-border-opacity));
```

What I’m trying to do here is to avoid hard-coding the color value coming from the theme.

I’m guessing this is a bug but I wanted to be sure I wasn’t overlooking something.

**Comment by voorjaar** [comment, marked as answer]
No. It's a bug. will fix this tomorrow.

**Reply by voorjaar** [reply]
https://github.com/windicss/windicss/issues/111

https://github.com/windicss/windicss/issues/112
